weblate/machinery/tests.py

```diff
@@ -248,6 +248,14 @@
 
 DEEPL_RESPONSE = {"translations": [{"detected_source_language": "EN", "text": "Hallo"}]}
 
+MICROSOFT_RESPONSE = [{"translations": [{"text": "Svět.", "to": "cs"}]}]
+
+MS_SUPPORTED_LANG_RESP = {
+    "translation": {'Item': 'data'},
+    "transliteration": {'Item': 'data'},
+    "dictionary": {'Item': 'data'},
+}
```

Comment:
You need to include real looking data, otherwise tests fail as the en cs translation seems unsupported.